x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_preview.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiEmptyPrompt, EuiFieldSearch, EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { EngineLogic } from '../engine';
+import { Result } from '../result/result';
+
+import { RelevanceTuningLogic } from '.';
+
+const emptyCallout = (
+  <EuiEmptyPrompt
+    data-test-subj="EnterAQueryPrompt"
```

Comment:
optional suggestion, but maybe something shorter and more general here instead of tying the var name too close to the copy - more future-proof if the copy changes

```suggestion
    data-test-subj="EmptyQueryPrompt"
```

Reply:
eaf97ea627a5df7056518d808680cd3994150129